app/assess/templates/macros/banner_summary.html

```diff
@@ -15,7 +15,7 @@ <h3 class="govuk-body-l fsd-banner-content">Total funding requested:
                         &pound;{{ "{:,.2f}".format(funding_amount_requested | float) }}</h3>
 
                     {% if user_role != "COMMENTER" %}
-                    {% if flag and flag.is_qa_complete and display_status not in ('FLAGGED', 'STOPPPED')  %}
+                    {% if flag and flag.is_qa_complete and display_status not in ('FLAGGED', 'STOPPED')  %}
```

Comment:
Was this the root cause of the issue, `STOPPPED`?

Reply:
Nope. Thought its a typo so corrected it. Saw the list of assessment status here `assessment_statuses = {
    "ALL": "All",
    "NOT_STARTED": "Not started",
    "IN_PROGRESS": "In progress",
    "COMPLETED": "Assessment complete",
    "QA_COMPLETED": "QA complete",
    "FLAGGED": "Flagged",
    "STOPPED": "Stopped",
}`

Reply:
Ah! Nice spot! Was this the root cause then: https://github.com/communitiesuk/funding-service-design-assessment/pull/216#pullrequestreview-1372579880